src/main/resources/config.properties

```diff
@@ -14,6 +14,7 @@ builder.matlab.customcommandoption.display.name = Custom
 Releaseinfo.matlab.version.not.found.error = Error finding MATLAB release for given MATLAB root. Verify MATLAB root path.
 matlab.not.found.error = Unable to launch MATLAB from the specified location. Verify the path to MATLAB root folder.
 matlab.not.found.error.for.node = Unable to launch MATLAB '%s' on the node '%s'. Verify global tool configuration for the specified node.
+matlab.execution.exception.prefix = Received non-zero exit code while trying to run MATLAB command (Exit code: %d)
```

Comment:
Received a nonzero exit code %d while trying to run MATLAB.

We use "run MATLAB" in the doc. For example:
https://www.mathworks.com/help/matlab/matlab_env/run-matlab-from-azure-marketplace.html?searchHighlight=%22run%20matlab%22&s_tid=srchtitle

If you want MATLAB to be followed by something, you might consider "code" or "commands".